xml/System.Security.Cryptography.Pkcs/Pkcs12ConfidentialityMode.xml

```diff
@@ -36,7 +36,7 @@
       </ReturnValue>
       <MemberValue>1</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The SafeContents value is not encrypted.</summary>
```

Comment:
Would this be better `P:System.Security.Cryptography.Pkcs.Pkcs12SafeContentsBag.SafeContents` ? Is this the correct one?
```suggestion
        <summary>The <see cref="P:System.Security.Cryptography.Pkcs.Pkcs12SafeContentsBag.SafeContents"/> value is not encrypted.</summary>
```

Reply:
Currently it's about the type Pkcs12SafeContents, not any particular property (the more likely place is the enumerator from Pkcs12Info).  I don't think the enum members should really reference another type directly in their documentation, because we almost certainly won't remember to come clean them up in a timely manner if the API scope increases (while maintaining the same concept).

Reply:
@bartonjs What about breaking the word so that it reads "safe contents" and therefor doesn't look like a specific property?

Reply:
It's referencing "SafeContents" data from RFC7292/PKCS#12.

If that seems too weird and we're trying to generalize it then "The data is not encrypted." is probably the best it'll get.